entity-service-impl/src/main/java/org/hypertrace/entity/query/service/converter/OrderByConverter.java

```diff
@@ -75,4 +89,8 @@ private static Map<org.hypertrace.entity.query.service.v1.SortOrder, SortOrder>
 
     return unmodifiableMap(map);
   }
+
+  private String getEncodedAlias(final String alias) {
```

Comment:
this is wrong actually. does it makes sense to encode all the sort fields? why do we encode `.` in functions but not other places?